emulsify.theme

```diff
@@ -4,3 +4,105 @@
  * @file
  * Functions to support theming.
  */
+
```

Comment:
This might make more sense to go in the [Emulsify Twig](https://github.com/emulsify-ds/emulsify_twig) module rather than in each theme. If it's going to be the recommended way to write libraries, I'd hate for someone to not know what this is, and delete it from their theme, and then ask us why their libraries don't work... If it's in the module (which is a Drupal dependency) they can't delete it, and keeps "required" code out of their custom codebase.

Reply:
To do that, we also need to change the code a little bit. (Need to get the active theme and dir)
https://github.com/emulsify-ds/emulsify-drupal/blob/79f34c5d45263045d19c29d0deb08ff4f01a5c10/emulsify.theme#L14

Or we can just wait for this feature to become included in the core https://www.drupal.org/project/drupal/issues/3092496

We can propagate the idea of `If you don't know and if all works good, DON'T touch anything!`  😄